src/serviceRegistry.js

```diff
@@ -1,30 +1,30 @@
 'use strict';
 
 const users = {
-    user_ms: {
+    userMicroService: {
```

Comment:
my style checker only receives camellCase, but this is only in the code, not in the deployment